apps/odyssey-stats/package.json

```diff
@@ -18,7 +18,7 @@
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"scripts": {
 		"clean": "npx rimraf dist",
-		"build": "NODE_ENV=production yarn dev",
+		"build": "NODE_ENV=production yarn dev && yarn translate",
```

Comment:
Any concerns that we're running `yarn translate` with `NODE_ENV=production`?

Reply:
Thanks for the review @tyxla !

Not really any concerns, we were just thinking if there are steps, we might just use them for different processes.